modules/travel_pay/app/services/travel_pay/claims_service.rb

```diff
@@ -21,6 +21,31 @@ def get_claims(params = {})
       }
     end
 
+    def get_claims_by_date_range(params = {})
+      if params['start_date'] && params['end_date']
```

Comment:
What are your thoughts on making end date optional and using `.now()` if empty? This is not something I necessarily want to see - just looking for your thoughts on this common pattern.

Reply:
🤔 I'm not opposed to it, depending on the use case. Was thinking this could take the place of the current "get all the claims then filter them for the target date" (to get the single claim for the single appt date). In that case we'd want to require the end date to be included (even if it's the same as the start date) vs. getting a slew of claims back... (Of course that approach would depend on how the migrations team builds out their endpoint, so that might be a moot point). I'm happy to discuss more if you'd like.

Reply:
That makes sense